server/chat.ts

```diff
@@ -1376,21 +1376,22 @@ export const Chat = new class {
 	 * Load chat filters
 	 *********************************************************/
 	readonly filters: ChatFilter[] = [];
-	filter(
-		context: CommandContext,
-		message: string,
-		user: User,
-		room: Room | null,
-		connection: Connection,
-		targetUser: User | null = null
-	) {
+	filter(context: CommandContext, message: string) {
```

Comment:
I think I'd prefer this the other way around: `message, context`.

Reply:
Okay.